.prettierignore

```diff
@@ -30,3 +30,6 @@
 LICENSE
 packages/**/dist/**
 yarn.lock
+
```

Comment:
based on the output from `prettier` I think we'll need to add `*.graphql` to this file